src/actions/actions.js

```diff
@@ -0,0 +1,7 @@
+export const CHANGE_THEME = 'CHANGE_THEME';
+export const SEARCH = 'SEARCH';
+export const SET_PARAM = 'SET_PARAM';
+export const SET_VIDEO = 'SET_VIDEO';
+export const SET_LIST_OF_VIDEOS = 'SET_LIST_OF_VIDEOS';
+export const SET_ERROR = 'SET_ERROR';
+export const SET_SEE_VIDEO_DETAIL = 'SET_SEE_VIDEO_DETAIL';
```

Comment:
Good, defining the actions this way makes things easy to maintain and less prone to mistyping the name of the actions